snooty.toml

```
@@ -25,10 +25,11 @@ atlas = "MongoDB Atlas"
 targets = ["motor.txt"]
 variant = "warning"
 value = """
-    As of May 14th, 2025, Motor has been sunset in favor of the GA release of the PyMongo Async API in Pymongo. \
-    No new features will be added to Motor. Motor will be deprecated on May 14th, 2026, \
-    and no further bug fixes will be released after deprecation, with the exception of critical bug fixes, which will be released until May 14th, 2027. \
-    We strongly recommend that Motor users migrate to PyMongo Async while Motor is still supported. \
-    To learn how to migrate from Motor to the PyMongo Async API, see the \
-    `Migrate to PyMongo Async guide <https://www.mongodb.com/docs/languages/python/pymongo-driver/current/reference/migration/>`__.
+    As of May 14, 2025, Motor is deprecated in favor of the GA release of the PyMongo Async
+    API in the PyMongo library. No new features will be added to Motor, and we will only
+    provide bug fixes until it reaches end of life on May 14, 2026. After that, we will only
+    fix critical bugs until final support ends on May 14, 2027. We
```

I: [only](https://www.mongodb.com/docs/meta/style-guide/terminology/alphabetical-terms/#o:~:text=Place%20only%20preceding%20the%20word%20or%20group%20that%20it%20modifies.%20If%20only%20modifies%20a%20final%20word%20or%20phrase%2C%20it%20may%20precede%20or%20follow%20that%20word%20or%20phrase.)

```suggestion
    API in the PyMongo library. No new features will be added to Motor, and we will
    provide only bug fixes until it reaches end of life on May 14, 2026. After that, we will
    fix only critical bugs until final support ends on May 14, 2027. We
```